tests/lib/managers/folders-test.js

```diff
@@ -276,4 +276,20 @@ describe('Folders', function() {
 		});
 	});
 
+	describe('deletePermanently()', function() {
+
+		it('should call BoxClient defaultResponseHandler method with the callback when response is returned', function(done) {
+			sandbox.mock(boxClientFake).expects('defaultResponseHandler').returns(done);
```

Comment:
We use `.withArgs()` to assert that the mocked function has the correct arguments passed to it.